business_rules/operators.py

```diff
@@ -860,13 +860,14 @@ def is_unique_set(self, other_value):
         target = self.replace_prefix(other_value.get("target"))
         comparator = other_value.get("comparator")
         values = [target, comparator]
-        target_data = flatten_list(values)
+        target_data = flatten_list(self.value, values)
```

Comment:
What's an example of `self.value` in this instance?

Reply:
@nhaydel  Okay I've gone back through this, so am I correct in saying that the use case I missed is below.

```
self.value = {
               "ARM": [["PLACEBO", "PLACEBO", "A", "A"], ["PLACEBO", "PLACEBO", "A", "A"]],
                "TAE": [1, 1, 1, 2],
                "LAE": [1, 2, 1, 2],
                "ARF": [1, 2, 3, 4],
            }
l = ["ARM", "LAE"]
```

If yes, is it worth me adding that case to the `test_is_unique_set` unit tests?

Reply:
I think so if they don't have tests already. And yes, self.value is a value of the type the operator is implemented for. For the dataframe type self.value is the pandas.Dataframe the rule is operating on